master/master.cfg

```diff
@@ -430,6 +435,67 @@ for branch_num, (git_url, branchname, git_branch) in enumerate(git_branches):
         ))
 
 
+# Set up Pull Request builders
+
+pull_request_builders = []
+
+for name, worker, buildfactory, stability in builders:
+    if stability != STABLE:
+        continue
+
+    if "Windows XP" in name or "VS9.0" in name:
+        continue
+
+    buildername = name + " " + "PullRequest"
```

Comment:
Just to bikeshed, I think I'd prefer just "PR".

Reply:
Amended in 222b16f